apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/ThreadFactoryImpl.java

```diff
@@ -8,6 +8,9 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import javax.enterprise.inject.Vetoed;
 
+import lombok.extern.slf4j.Slf4j;
+
```

Comment:
Was that class removed ?

Reply:
There is a new NamedThreadFactory class in the 'task' package.